webonise5/src/CollectionAndStringOperationApp.java

```diff
@@ -0,0 +1,37 @@
+import services.CollectionService;
+import services.StringOperationsService;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class CollectionAndStringOperationApp {
+
+    private static CollectionService collectionService = new CollectionService();
+    private static StringOperationsService stringOperationService = new StringOperationsService();
+    private static Scanner scan = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        //Collection operation starts....
+        ArrayList<Object> inputList = new ArrayList();
+        HashMap<Object, Object> inputMap = new HashMap();
```

Comment:
why use object as key, value type?
this eliminates restriction on your input and makes error handling difficult.